slam-libraries/viam-cartographer/src/io/submap_painter.cc

```diff
@@ -113,5 +113,27 @@ cartographer::io::PaintSubmapSlicesResult PaintSubmapSlices(
                                                      origin);
 }
 
+// Draw a position on the current map
```

Comment:
[nit] Use pose instead of position.
[opt] Suggestion: "Draw a marker for the current pose within the map"